Bio/Blast/Applications.py

```diff
@@ -1235,7 +1235,8 @@ def __init__(self, cmd="deltablast", **kwargs):
 
 
 def _test():
-    """Run the Bio.Blast.Applications module's doctests."""
+    """Run the Bio.Blast.Applications module's doctests.(PRIVATE)."""
+
```

Comment:
You need a space not a full stop before the ``(PRIVATE)`` text. Also remove the extra blank link which will cause one of the style checks to fail.